integration/src/womir_translation.rs

```diff
@@ -1760,7 +1760,8 @@ impl<'a, F: PrimeField32> Settings<'a> for OpenVMSettings<F> {
                     .collect()
             }
 
-            _ => todo!(),
+            Op::Nop => vec![Directive::Instruction(ib::addi(0, 0, F::ZERO))],
```

Comment:
this doesn't break WOM semantically, but does break it syntactically. It'd be good to have a better solution for this

Reply:
actually adjusted the tests, nop not needed